index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
```

Comment:
This works but since ES6, let and const are preferred for declaring variables.  As a rule of thumb in this course, never use var.  If you use let, JS will let you mutate or change the variable to something else. 
let dog = "maizey"
dog.toUpperCase()  (mutating the string to upper case)
dog = "Chase" (saving a different string to the same variable)
dog = [1,2,3,4] (saving an array to the same variable)

If you use const, you can mutate what is saved but you cannot change it
dog.toUpperCase()  (mutating the string to upper case. mutation is allowed with const)
dog = "Chase" (this would cause an error)

Mutation is taking the existing item and changing it a little such as adding or removing an item from array

Once you use const to declare a variable, you cannot use "=" with it again or you will get an error. Try playing around with this in the console in your browser.